base/task.jl

```diff
@@ -542,13 +542,13 @@ true
 """
 function schedule(t::Task, @nospecialize(arg); error=false)
     # schedule a task to be (re)started with the given value or exception
-    t.state == :runnable || Base.error("schedule: Task not runnable")
+    t.state === :runnable || Base.error("schedule: Task not runnable")
     if error
         t.queue === nothing || Base.list_deletefirst!(t.queue, t)
-        t.exception = arg
+        setfield!(t, :exception, arg)
```

Comment:
Why does this help? Does somebody overwrite a setproperty that generic?

Reply:
It's the conversion to `Any` in setproperty.